--- picitem.cpp	2009-02-08 15:47:03.000000000 +0200
+++ /tmp/qt3/src/picitem.cpp.patched	2016-09-27 15:46:34.320539908 +0300
@@ -131,17 +131,17 @@
 	if ( (m_pinSettings->type() == PinSettings::pt_input && onLeft) ||
 			 (m_pinSettings->type() == PinSettings::pt_output && !onLeft) )
 	{
-		pa[0] = QPoint::QPoint( midRight, midY );
-		pa[1] = QPoint::QPoint( midLeft, midY - PinDirArrow );
-		pa[2] = QPoint::QPoint( midLeft, midY + PinDirArrow );
+		pa[0] = QPoint( midRight, midY );
+		pa[1] = QPoint( midLeft, midY - PinDirArrow );
+		pa[2] = QPoint( midLeft, midY + PinDirArrow );
 		p.drawPolygon(pa);
 		p.drawLine ( left, midY, right, midY );
 	}
 	else // Left facing arrow
 	{
-		pa[0] = QPoint::QPoint( midLeft, midY );
-		pa[1] = QPoint::QPoint( midRight, midY - PinDirArrow );
-		pa[2] = QPoint::QPoint( midRight, midY + PinDirArrow );
+		pa[0] = QPoint( midLeft, midY );
+		pa[1] = QPoint( midRight, midY - PinDirArrow );
+		pa[2] = QPoint( midRight, midY + PinDirArrow );
 		p.drawPolygon(pa);
 		p.drawLine ( left, midY, right, midY );
 	}
